src/tools/info/type_info.c

```diff
@@ -87,7 +87,7 @@ static void print_size(const char *name, size_t size)
     print_size(UCS_PP_QUOTE(_type) "." UCS_PP_QUOTE(_field), \
                ucs_field_sizeof(_type, _field))
 
-void print_type_info(const char * tl_name)
+static void print_ucs(const char *tl_name)
```

Comment:
Just curious - making `print_size()` non-inline won't solve stack issue here?

Reply:
It was the first thing I tried, I think it's related to the excessive macro calling

Reply:
To reproduce this you'll need to run the full CI here: https://github.com/openucx/ucx/pull/9963 without this commit https://github.com/openucx/ucx/pull/9963/commits/2388f0db1d1e710b3a69adab3a0526ea9e734f63
@Artemy-Mellanox

Reply:
@michal-shalev can you pls copy/paste the CI failure in print_type_info without this fix?

Reply:
I couldn't reproduce it, but it failed in the CI, I'll document these cases in the future.
@Artemy-Mellanox @yosefe